packages/host/tests/acceptance/theme-card-test.gts

```diff
@@ -180,84 +241,306 @@ module('Acceptance | theme-card-test', function (hooks) {
+              },
+            },
+            type: 'card',
+            attributes: {
+              cardInfo: {
+                title: 'Soft Pop',
+                description: 'A theme with soft color pops',
+              },
+            },
+          },
+        },
       },
     });
   });
 
-  test('renders structured theme card with inline css variables', async function (assert) {
-    assert.expect(11);
-    await visitOperatorMode({
-      stacks: [[{ id: themeCardId, format: 'isolated' }]],
+  module('style-reference-card', () => {
+    test('renders with inline css variables', async function (assert) {
+      assert.expect(5);
+      await visitOperatorMode({
+        stacks: [[{ id: styleRefCardId, format: 'isolated' }]],
+      });
+      assert
+        .dom(`[data-test-card="${styleRefCardId}"] h1`)
+        .hasText('Starry Night');
+      assert
+        .dom(`[data-test-card="${styleRefCardId}"]`)
+        .hasClass('boxel-card-container--themed');
+      assert
+        .dom(`[data-test-card="${styleRefCardId}"]`)
+        .hasAttribute('style', ROOT_STYLE_ATTRS);
+
+      let container = document.querySelector<HTMLElement>(
+        `[data-test-card="${styleRefCardId}"]`,
+      );
+      assert.ok(container, 'theme card container is present');
+      let computedFontFamily = window
+        .getComputedStyle(container!)
+        .getPropertyValue('font-family');
+      assert.ok(
+        computedFontFamily.includes('Libre Baskerville'),
+        `computed font-family includes themed value`,
+      );
     });
-    assert.dom(`[data-test-card="${themeCardId}"] h1`).hasText('Starry Night');
-    assert.dom('[data-test-css-vars]').containsText('--radius: 0.75rem;');
-    assert
-      .dom(`[data-test-card="${themeCardId}"]`)
-      .hasClass('boxel-card-container--themed');
-    assert
-      .dom(`[data-test-card="${themeCardId}"]`)
-      .hasAttribute('style', ROOT_STYLE_ATTRS);
-
-    let container = document.querySelector<HTMLElement>(
-      `[data-test-card="${themeCardId}"]`,
-    );
-    assert.ok(container, 'theme card container is present');
-    let styleAttr = container?.getAttribute('style') ?? '';
-    assert.ok(
-      styleAttr.includes('--background: #0a0f23'),
-      'inline style includes root background variable',
-    );
-    assert.false(
-      styleAttr.includes('--background: #050813'),
-      'dark mode value is not applied inline',
-    );
-    let computedFontFamily = window
-      .getComputedStyle(container!)
-      .getPropertyValue('font-family');
-    assert.ok(
-      computedFontFamily.includes('Libre Baskerville'),
-      `computed font-family includes themed value`,
-    );
-
-    await visitOperatorMode({
-      stacks: [],
-      codePath: themeCardId,
-      submode: 'code',
+  });
+
+  module('structured-theme-card', () => {
+    test('renders with inline css variables', async function (assert) {
+      assert.expect(11);
```

Comment:
Is this needed for this test and the one above? I thought it only was for tests where assertions happen in a promise handler or the like, like `onSave` in the next one